afm/logging.py

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
Should we perhaps put this file in fybrik/fybrik/pkg/logging so that all python based fybrik components can use it?